AWSSDKSwiftCLI/Sources/AWSSDKSwiftCLI/Models/PackageManifestBuilder.swift

```diff
@@ -123,6 +131,10 @@ struct PackageManifestBuilder {
         "let excludeRuntimeUnitTests = \(excludeRuntimeTests)"
     }
 
+    private func buildPreviewFlag() -> String {
+        "let isPreviewBuild = \(previewBuild)"
+    }
+
```

Comment:
A simple boolean variable is rendered into the `Package.swift`.  It is `true` when the CLI flag is set.